### ISSUE TITLE
Finalize selected wireframes in collaboration with business area

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#225]
#### Describe the task

We need to collaborate with IRC and finalize the certain screens, through the spreadsheet and within the wireframe. Collaboration will happen in Giraffe Collab as well as through the SME channel. Notes will be captured and shared with the group. There may be need to collab with Moose and this ticket can get quite big

#### Acceptance Criteria

- [x] Verify fields for Production Data
- [x] Verify workflow for supplementary/corrected reports with John
- [x] Verify fields for Non-attributable emissions
- [x] Verify fields for captured emissions
- [x] Verify fields for Operation information (part 1 & 2)
- [x] Verify fields for facility information
- [ ] _Clarify Allocations of Emissions - 100%, where to display this (emissions summary?)_
- [ ] _Verify fields for verification_
- [x] Verify fields for Emissions Summary
- [x] Verify fields for Annual report section 2
- [x] verify LFO report data

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by suhafa** [comment]
'Verify workflow for supplementary/corrected reports with John' completed as part of #176

**Comment by suhafa** [comment]
Need to verify BCGHG ID column needed or not in reporting
Edit: Ruled out having this, can bring up in user engagement if users will benefit from this

**Comment by suhafa** [comment]
Updated Person responsible, Non-attributable emissions, Captured emissions, production data wireframe. Waiting on BA to confirm the wireframe implementation

**Comment by suhafa** [comment]
Verification - This has been put on hold until we verification templates are made.

**Comment by suhafa** [comment]
Allocation of emissions is also put on hold till we receive direction from IRC about where it will be. LFO Report Data has been converted to Operation Reporting Data.

**Comment by patriciarussellCAS** [comment]
@suhafa I am prepping the emails now so will close this ticket.
